tests/test_edgeql_syntax.py

```diff
@@ -948,28 +957,28 @@ def test_edgeql_syntax_name_22(self):
         """
 
     @tb.must_fail(errors.EdgeQLSyntaxError,
-                  r'Identifiers cannot contain "::"', line=2, col=16)
+                  'backtick-quoted name can\'t contain `::`', line=2, col=16)
```

Comment:
Please use "cannot" instead of "can't", we avoid the use of contractions in error messages.

Reply:
Fixed